api/v1/listing.py

```diff
@@ -23,6 +49,41 @@ def get(self):
         else:
             return response, 200
 
+    @swagger.operation(
+        notes='Add a new listing',
+        parameters=[
+            {
+                "name": "listing",
+                "dataType": "string",
```

Comment:
we should probably note that we expect this hash to have been created with a `keccak256` equivalent